### PR TITLE
Update external libraries versions and improve whoami message

### LINKED ISSUE #49: Update versions of external libs of workflows
The problem is:
image
image
image

Need to update libraries versions.

### PR DESCRIPTION
**Pull request by Ivanio1**
Closed #49

### CODE REVIEW

**Comment by Ivanio1** [comment]
@ryukzak

**Comment by Ivanio1** [comment]
@ryukzak waiting for merging and ready to open new MR - improving whoami command

**Comment by Ivanio1** [comment]
Closed #46

**Comment by Ivanio1** [comment]
@ryukzak

File: src/course_bot/general.clj
Comment (ryukzak) [inline src/course_bot/general.clj:130]: Old formatting is better.

**Review by ryukzak — CHANGES_REQUESTED** [review]
1. Please, separate formatting changes from actual changes to different PRs.
2. Such reformatting I can approve only if it is a result of autoformatter works (so it should be checked by CI). So it should be for whole project or bot applied.

```clj
     :restart-wrong-input           "Неправильный ввод. Ожидается: /restart 12345"
     :restarted-and-notified        "Перезапущено и уведомлено: "
     :use-start-once-more           "Вы можете использовать /start еще раз."
```

3. Please, rewrite connected test in this way: https://github.com/ryukzak/course-bot/pull/55/files

File: src/course_bot/general.clj
Comment (ryukzak) [inline src/course_bot/general.clj:88]:
```suggestion
         grouplab1                                    (-> pres :lab1 :group)]
```

Comment (ryukzak) [inline src/course_bot/general.clj:88]: In theory, csa.clj can specify several presentations for the course. So we need to pass it as an argument to `whoami-talk` command. Like:

```clj
      (general/whoami-talk db conf (pres/group "lab1") (pres/group "labN"))
```

The output of `pres/group` should be connected to the end of normal whoami messages

**Review by ryukzak — CHANGES_REQUESTED** [review]

**Comment by Ivanio1** [comment]
@ryukzak Okay, I will close this PR, then open First PR about updating versions. And after merging I will continue to work with whoami.

**Pull request closed without merging**
